CMakeLists.txt

```diff
@@ -46,7 +46,7 @@
 #
 # ============================================================================
 
-cmake_minimum_required (VERSION 2.8)
```

Comment:
This was on the backburner for some time but I've updated travis to use a newer CMake version. 

In the issue you commented on that the following was needed to remove the possibly weird behaviour
```
cmake_policy(SET CMP0054 NEW)
```

Shouldn't that be part of this pull request too?